src/common/utils/helpers.ts

```diff
@@ -82,3 +87,8 @@ export const refSchema = z
   .string()
   .optional()
   .default(process.env.DEFAULT_REFERENCE || 'Default_Reference_Name');
+
+export const resetTokenSchema = z
```

Comment:
why a token has a schema?

Reply:
yeah it's redundant, we'll use jwt function to check the token's validity. I will remove the schema here.